packages/api/cms-api/src/file-uploads/file-uploads.controller.ts

```diff
@@ -29,6 +41,31 @@ export function createFileUploadsController(options: { public: boolean }): Type<
 
             return fileUpload;
         }
+
+        @Get("download/:id")
+        @RequiredPermission(["fileUploads"], { skipScopeCheck: true })
+        async downloadFileById(@Param("id") id: string, @Res() res: Response): Promise<void> {
```

Comment:
Do we really want to add this endpoint in this PR? Didn't we discuss adding this with a secret, timeout etc.?

Reply:
Removed it for now, as discussed internally: ee1944c8bacdb77d8d378675db09445bd6ceed65